pyproject.toml

```diff
@@ -65,7 +65,7 @@ dependencies = [
 dynamic = ["version"]
 
 [project.optional-dependencies]
-torchvision = ["torchvision >=0.20.dev,<0.21.0"]
+# torchvision = ["torchvision >=0.20.dev,<0.21.0"] # Causes a bunch of issues with uv sync and tries to force install of a torch from June
```

Comment:
seems like this issue is only related to uv.
when use pip it is not a iisue.

Reply:
when you use pip doesnt look at torchvision until you ask for the torchvision extra dep, I think thats the issue